main.js

```diff
@@ -0,0 +1,540 @@
+					this.log('debug', `Message sent`)
+					const value = await context.parseVariablesInString(action.options.data)
+					if (this.config.debug_messages) {
+						this.log('debug', `Message sent: ${value}`)
+					}
+          if (typeof(action.options.callPlan) === 'number') {
+            this.ws.send(JSON.stringify({
+              KommanderMsg: "KommanderMsg_IndexInvokePrePlan",
+              params: {
+                index: action.options.callPlan - 1,
+                onlySetReal: true
+              }
+            }))
+          }else{
+            this.ws.send(JSON.stringify({
+              KommanderMsg: "KommanderMsg_PrePlanNextOrPre",
+              params: {
+                next: action.options.callPlan === 'NextPlan', // true下一条
+              }
+            }))
+          }
+				},
+			},
+      ChangePlanGroup: {
+				name: 'ChangePlanGroup',
+				options: [
+					{
+						id: 'changePlanGroup',
+						type: 'dropdown',
+						label: 'next',
+						choices: changePlanGroupDropDownValue,
+						default: 'next',
+					},
+				],
+				callback: async (action, context) => {
+					this.log('debug', `Message sent`)
+					const value = await context.parseVariablesInString(action.options.data)
+					if (this.config.debug_messages) {
+						this.log('debug', `Message sent: ${value}`)
+					}
+          if (typeof(action.options.changePlanGroup) === 'number') {
+            this.ws.send(JSON.stringify({
+              KommanderMsg: "KommanderMsg_SwitchPlanPreGroup",
+              params: {
+                index: action.options.changePlanGroup - 1,
+              }
+            }))
+          }else{
+            this.ws.send(JSON.stringify({
+              KommanderMsg: "KommanderMsg_SwitchPlanPreGroup",
+              params: {
+                type: action.options.changePlanGroup,
+              }
+            }))
+          }
+				},
+			},
+			ChangePlayStatus: {
+				name: 'ChangePlayStatus',
+				options: [
+					{
+						id: 'changePlayStatus',
+						type: 'dropdown',
+						label: 'ChangePlayStatus',
+						choices: [
+							{ id: 'Play', label: 'Play' },
+							{ id: 'Pause', label: 'Pause' },
+							{ id: 'Stop', label: "Stop"}
+						],
+						default: 'Play',
+					},
+				],
+				callback: async (action, context) => {
+					this.log('debug', `Message sent`)
+					const value = await context.parseVariablesInString(action.options.data)
+					if (this.config.debug_messages) {
+						this.log('debug', `Message sent: ${value}`)
+					}
+					this.ws.send(JSON.stringify({
+            KommanderMsg: `KommanderMsg_${action.options.changePlayStatus}`,
+            params: {
+              onlySetReal: true
+            }
+          }))
+				},
+			},
+      SoundControl: {
+				name: 'SoundControl',
+				options: [
+					{
+						id: 'soundControl',
+						type: 'dropdown',
+						label: 'SoundControl',
+						choices: [
+              { id: 0, label: "Mute/Unmute"},
+							{ id: -1, label: 'volume+' },
```

Comment:
Please check for consistent uppercase/lowercase writing, e.g. `Volume +` would be better here.